### DISCUSSION TITLE
책 수요조사: 프로그래머의 길, 멘토에게 묻다(Apprenticeship Patterns: Guidance for the Aspiring Software Craftsman)

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#135] [General]
제가 알기론 저와 오래 얘기하신 몇 분은 책이 있는 걸로 알고 있는데
책이 없으신 분은 이번에도 책을 선물해 드리도록 하겠습니다.

책이 없어서 저에게 받으실 분은 댓글을 달아 주시면 
내일인 2월 9일 수요일 까지 취합 후
2월 10일 목요일에 주문해서
2월 11일 금요일에 배송될 수 있도록
몰래 배송 주소를 물어보겠습니다.

**Comment by wooyaggo86** [comment]
혹시 저도... ;ㅅ;

**Reply by jongfeel** [reply]
가능합니다!

**Comment by jongfeel** [comment]
!?

다들 책이 있거나 알아서 구매하시는 걸로 이해해야 겠군요.

저의 용돈을 아껴 주셔서 감사합니다!

**Reply by TaeHyoungKwon** [reply]
넵넵 책 있어서 신청안했습니다 😄

**Reply by hi-hyein** [reply]
넵 저두요!

**Comment by momoci99** [comment]
아아아앗 하루 늦었네요 사비로 샀습니다 하핳

**Comment by Joseph-Cha** [comment]
엌.. 저 늦었지만 신청이 가능할까요 허허허..

**Reply by jongfeel** [reply]
늦었지만 전달은 해 드리겠습니다.

**Comment by evelon** [comment]
엄청 늦게 보았지만 찾아보니 집현전에 책이 있군요 ㅎㅎ
